pkgs/development/r-modules/wrapper.nix

```diff
@@ -1,9 +1,16 @@
-{ stdenv, R, makeWrapper, recommendedPackages, packages }:
+{ stdenv, R, libcxx, makeWrapper, recommendedPackages, packages
+
+, Cocoa, Foundation
+}:
 
 stdenv.mkDerivation {
   name = R.name + "-wrapper";
 
-  buildInputs = [makeWrapper R] ++ recommendedPackages ++ packages;
+  buildInputs = [makeWrapper R] ++ recommendedPackages ++ packages
+    ++ stdenv.lib.optionals stdenv.isDarwin [ Cocoa Foundation ];
```

Comment:
Is it not possible to register those paths in *R* rather than duplicating them in both the generic builder and the wrapper?